test/e2e.contract.events.js

```diff
@@ -27,6 +28,9 @@ describe('contract.events [ @E2E ]', function() {
     })
 
     it('contract.getPastEvents', async function(){
+        // `getPastEvents` not working with Geth instamine over websockets.
+        if (process.env.GETH_INSTAMINE) return;
```

Comment:
Caught this - `getPastEvents`
**Works**
+ ganache (http/ws)
+ geth instamine (http)
+ geth automine (http/ws)

**Broken**
+ geth instamine (ws)

Just skipping this case because it's probably something weird about Geth and not web3's fault

Reply:
Made this  more general so geth.instamine doesn't run ws tests - it's really flaky.